treeherder/webapp/api/resultset.py

```diff
@@ -209,13 +209,14 @@ def trigger_runnable_jobs(self, request, project, jm, pk=None):
             return Response({"message": "No resultset with id: {0}".format(pk)},
                             status=HTTP_404_NOT_FOUND)
 
-        buildernames = request.data.get('buildernames', [])
-        if len(buildernames) == 0:
-            Response({"message": "The list of buildernames cannot be empty"},
+        requested_jobs = request.data.get('requested_jobs', [])
+        decisionTaskID = request.data.get('decisionTaskID', [])
+        if len(requested_jobs) == 0:
```

Comment:
I know this was pre-existing, but this can be written as just `if requested_jobs:` :-)